package/opt/hassbian/suites/homebridge.sh

```diff
@@ -128,20 +128,22 @@ fi
 
 echo "Checking the installation..."
 validation=$(pgrep -f homebridge)
-if [ "$validation" != NULL ]; then
-	echo
-	echo -e "\e[32mInstallation done.\e[0m"
-	echo
-	echo "Homebridge is now running and you can add it to your"
-	echo "home-kit app on your iOS device, when you are asked for a pin"
-	echo "use this: '$HOMEBRIDGE_PIN'"
-	echo "For more information see this repo:"
-	echo "https://github.com/home-assistant/homebridge-homeassistant#customization"
-	echo
+if [ ! -z "${validation}" ]; then
+  echo
+  echo -e "\\e[32mInstallation done.\\e[0m"
+  echo
+  echo "Homebridge is now running and you can add it to your"
+  echo "home-kit app on your iOS device, when you are asked for a pin"
```

Comment:
Isn't the app called HomeKit?

Reference: https://www.apple.com/shop/accessories/all-accessories/homekit

Reply:
It its :)